src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -197,17 +197,12 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
     return filters.reduce((items, filter) => filter(items), items);
   }
 
-  @computed get allItems() {
```

Comment:
Isn't removing this technically a breaking change?

Reply:
Why? 
allItems was used in one place before the change and contained just one function which, in fact, was moved to after the  data source selecting

Reply:
@pashevskii this component is exported via extension API. Which means we don't know who uses this.

Reply:
Probably this isn't a breaking change since `allItems` method is used internally inside component. You can't use it outside, you have to only pass a props listed.

Reply:
Oh right. I thought it was basically the same issue as with `filterItems` (https://github.com/lensapp/lens/pull/2123#discussion_r574485966)
but that was leaking through `ItemListLayoutProps`. `allItems` is not there